README.md

```diff
@@ -159,58 +159,54 @@ automatically so that users don't have to worry about it.
 
 Next, we
 [start](./src/jit.rs#L132)
-an initial extended basic block (EBB), which is the entry block of the function, and
+an initial basic block (block), which is the entry block of the function, and
 the place where we'll insert some code.
 
  - A basic block is a sequence of IR instructions which have a single entry
    point, and no branches until the very end, so execution always starts at the
    top and proceeds straight through to the end.
 
- - Cranelift IR uses *extended* basic blocks, which are similar to basic blocks,
-   except they may branch out from the middle.
-
-Cranelift's extended basic blocks can have parameters. These take the place of
+Cranelift's basic blocks can have parameters. These take the place of
 PHI functions in other IRs.
 
-Here's an example of an EBB, showing branches (`brif`) that may leave the EBB in
-the middle of the instruction sequence, and demonstrating some EBB parameters.
+Here's an example of a block, showing branches (`brif` and `jump`) that are at the end of the block,
+and demonstrating some block parameters.
 
 ```
-ebb0(v0: i32, v1: i32, v2: i32, v507: i64):
+block0(v0: i32, v1: i32, v2: i32, v507: i64):
     v508 = iconst.i32 0
     v509 = iconst.i64 0
     v404 = ifcmp_imm v2, 0
-    brif eq v404, ebb8(v0, v1, v2)
     v10 = iadd_imm v2, -7
     v405 = ifcmp_imm v2, 7
-    brif ugt v405, ebb29(v10)
-    jump ebb29(v508)
+    brif ugt v405, block29(v10)
+    jump block29(v508)
 ```
 
 The `FunctionBuilder` library will take care of
-inserting EBB parameters automatically, so frontends that don't need to use
+inserting block parameters automatically, so frontends that don't need to use
 them directly generally don't need to worry about them, though one place they
 do come up is that incoming arguments to a function are represented as
-EBB parameters to the entry block. We must tell Cranelift to add the parameters,
+block parameters to the entry block. We must tell Cranelift to add the parameters,
 using
-[`append_ebb_params_for_function_params`](https://docs.rs/cranelift-frontend/0.25.0/cranelift_frontend/struct.FunctionBuilder.html#method.append_ebb_params_for_function_params)
+[`append_block_params_for_function_params`](https://docs.rs/cranelift-frontend/0.25.0/cranelift_frontend/struct.FunctionBuilder.html#method.append_block_params_for_function_params)
```

Comment:
nit: can you update the version number? (or is there a way to have a docs link that's generic with respect to the version?)

Reply:
It is possible to use `*` as version number.

Reply:
Also we can use `latest` as version number.  I'd like to use `latest` because of its explicitness.